aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -116,6 +137,18 @@ public void testNewOutputStream() throws Exception {
     Assert.assertEquals(content, result);
   }
 
+  @Test
+  public void testNewOutputStreamWithAccessPoint() throws Exception {
```

Comment:
Probably no need to test those, but I think this should work as well for multi-region access point and bucket alias, right?

Reply:
yes this should work out of box, tested mrap locally it works